docs/source/ci-cd.md

```diff
@@ -187,6 +188,85 @@ pipelines:
 ```
 
 
+## Jenkins
+To set up `rover` for use with Jenkins, you'll first want to consider what type of Jenkins `agent` you'll be using in your pipelines. The below samples are showcasing a golang pipeline utilizing Docker, but can be extended to meet your specific needs. 
+
+### Pipelines using Distributed Builds via Node Agent
+If you're running a distributed build system using Jenkins's `node` agent type, you'll need to ensure that `rover` is installed on all machines as part of a baseline image, or via a setup script, and available via a `PATH` variable globally. 
+
+### Pipelines using Docker
+When using Docker containers as part of your build process, you'll need to be aware of a few additional things. Normally when installing, Rover adds the path of its executable to your `$PATH`. However, Jenkins doesn't persist the `$PATH` variable between runs of `sh`  `steps` as each `sh` block is run as its own process. This means that if you install Rover and try to run it in the next step, you get a `command not found: rover` error. This is functionally similar to [CircleCI note](#linux-jobs-using-the-curl-installer), however with a different solve.
```

Comment:
```suggestion
If you're using Rover with a Docker-enabled pipeline, note the following additional considerations:

#### `$PATH` issues

[put path stuff here]

#### Permission issues

[put permission stuff here]
```

Reply:
Don't commit this suggestion directly, but does it make sense as far as splitting out the stuff in this section?

Reply:
Done!